s3torchconnectorclient/python/tst/integration/test_mountpoint_s3_integration.py

```diff
@@ -404,6 +404,44 @@ def test_delete_object_invalid_bucket(
         )
 
 
+@pytest.mark.parametrize("force_path_style", [False, True])
+def test_copy_object(sample_directory: BucketPrefixFixture, force_path_style: bool):
```

Comment:
No test for using the object info returned. One should be added, or the return type should be changed

Reply:
Will return `None` instead